examples/notebooks/Emukit-how-to-write-a-notebook.ipynb

```diff
@@ -4,28 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<center>\n",
-    "    <h1>\n",
-    "        Title of notebook ... with <b style=\"color:#FF9900\">Emukit</b>\n",
-    "    </h1>\n",
-    "    <br>\n",
-    "    <hr>\n",
-    "    <br>\n",
-    "<center>\n",
-    "    By *Author 1* and *Author 2* with edits by *Editor 1*\n",
-    "    \n",
-    "    Place, date"
+    "# Title of notebook (... with Emukit)"
```

Comment:
why brackets around "... with Emukit?"

Reply:
To indicate this is optional. Maybe I should remove " (... with Emukit)" altogether?